ddtrace/contrib/internal/logging/patch.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, trace_id: int, span_id: int, service: str, version: str, env:
 def _w_makeRecord(func, instance, args, kwargs):
     # Get the LogRecord instance for this log
     record = func(*args, **kwargs)
-    if config._logs_injection != LogInjectionState.ENABLED:
+    if config._logs_injection is LogInjectionState.DISABLED:
```

Comment:
Actually I think we need to update this line and line 64 to:

if config._logs_injection i LogInjectionState.ENABLED:
```suggestion
    if config._logs_injection in (LogInjectionState.DISABLED, LogInjectionState.STRUCTURED):
```

Or update this block to:

```
if config._logs_injection is LogInjectionState.DISABLED:
Comment
Actually I think we need to update this line and line 64 to:

```suggestion
    if config._logs_injection in (LogInjectionState.DISABLED, LogInjectionState.STRUCTURED):
```
Or update both `_w_StrFormatStyle_format` and `_w_makeRecord` to do nothing if `LogInjectionState` is not ENABLED. Something like

```
    if config._logs_injection is LogInjectionState.ENABLED:
        record.__dict__.update(ddtrace.tracer.get_log_correlation_context())
    return record
```

Reply:
The main issue is that the logging integration is now enabled by default: https://github.com/DataDog/dd-trace-py/blob/573a530416a3ed272704668fb99e74205cdfa91f/ddtrace/_monkey.py#L96.